src/patternfly/demos/NotificationDrawer/notification-drawer-template.hbs

```diff
@@ -11,7 +11,7 @@
   {{#> page-drawer}}
     {{#> drawer}}
       {{#> drawer-main}}
-        {{#> drawer-content}}
+        {{#> drawer-content drawer-content--NoBody=true}}
```

Comment:
Just removes `drawer__body`, so `page__main-container` is a child of `drawer__content`, a flex column. Allows us to manage the height/alignment of `page__main-container` better